### DISCUSSION TITLE
Broken CSS

### DISCUSSION THREAD

**Discussion by kyungnapark** [thegetty/quire#166] [Get Help] [answered]
Hi all,
My issue is that the css is broken every time I try to preview my project. When I inspect the page, it shows these 2 errors.
![Screen Shot 2021-08-26 at 9 55 52 AM](https://user-images.githubusercontent.com/87673056/131008023-5f70a48c-6caa-4594-8e7e-8481233f45ce.png)
I've also attached a screenshot of the what the site looks like.
![Screen Shot 2021-08-26 at 10 26 44 AM](https://user-images.githubusercontent.com/87673056/131008629-157400ed-5268-4f19-b3ed-b0c603afd7a4.png)

**Comment by geealbers** [comment, marked as answer]
Hi! Had you recently changed a color variable in colors.scss or variables.scss? I think the error you're seeing is because there's either an invalid color name somewhere, or because there's some formatting around a color name that's wrong, like a missing semicolon after it or something.

**Reply by kyungnapark** [reply]
I was missing a $ in $off-white. Thanks Greg!
